README.md

```diff
@@ -95,15 +91,14 @@ The following command will scan the directory `$TXT_DIR` looking for .txt files
 mvn exec:java -Dexec.mainClass=phrasecount.cmd.Load -Dexec.args="fluo.properties $TXT_DIR" -Dexec.classpathScope=test
```

Comment:
I like how simple these commands are but they could be put in a bash script (like `load.sh <textDir>`) to make it even easier for users. Otherwise, they have to copy, paste, and edit the commands.  The bash script could also check for inputs and print usage.